src/cat_generator.js

```diff
@@ -0,0 +1,13 @@
e Obama', 'Mig E Smalls', 'Milkshake', 'Milkshake', 'Milli', 'Minnie Pearl', 'Miracle Max', 'Moa', 'Moby', 'Mohammed Ali', 'mojo Nixon', 'Molder', 'Molly Ringworm', 'Momo', 'Mornigstar', 'Mortitia Adams', 'Morty', 'Mothra', 'Moxie', 'Mr. Kincade', 'Mr. Pants', 'Mr. Pibb', 'Mr. Pincess', 'Mr. Sunshine', 'Mr. T', 'Mrs Landingham', 'Mrs. Buttersworth', 'Mug Shot', 'Muggle', 'Mustard', 'Nacho', 'Nagini', 'Nahla', 'Nancy Reagon', 'Nathan Detroit', 'Neil Patrick Harris', 'Nelson Mandella', 'Nero', 'Neville', 'Nih', 'Nikola Tesla', 'Noodle', 'Nut Loaf', 'Nuttella', 'Oden', 'Omlette', 'Ophelia', 'Opuurah', 'Orca', 'Orchid', 'Ori', 'Orion', 'Orlando', 'Pancake', 'Panchetta', 'Panda', 'Panko', 'Pantalones', 'Paris', 'Paris', 'Patntho', 'Paw Revere', 'Peaches', 'PEanut', 'Pear', 'Peecasso', 'Penne', 'Peppermint', 'Persephonie', 'Petal', 'Peter Sellers', 'Phinneaus', 'Pickle', 'Pickles', 'Piggly Wiggly', 'Pingo', 'Pinko', 'Pinot Noir', 'Pippi Longstocking', 'Pistil', 'Pocahontis', 'Pocket', 'Popcorn', 'Porky', 'Portions', 'Praline Pecan', 'Pringle', 'Professor Pants', 'Pudding', 'Puddle', 'Pugnolia', 'Pumpernickle', 'Pumpkin Spice', 'Purrminie', 'Pussy Willow', 'Puursstant', 'Pythgerus', 'Queen Bea', 'Radagast', 'Ragon', 'Rah jah', 'Ralph Nader', 'Raymond', 'Rebel Yell', 'Regan', 'Ricardo Maontibhan', 'Rick James', 'Rigatoni', 'Rocket', 'Rockmoninoaf', 'Rockshasa', 'Romeo', 'Ron Swanson', 'Ronkonkama', 'Rose', 'Rosenkrantz', 'Ross Perot', 'Rubella', 'Rum Tum Tuggle', 'Rumbold', 'salvadore meowi', 'San Louis Abispo', 'Sandy', 'Sangria', 'Sarah brightman', 'Sarah Lee', 'Sashime', 'Savage', 'Scooter', 'Scotty', 'Scrappy', 'Scrimshaw', 'Scuba', 'Seabiscuit', 'Secriatriat', 'Selena', 'Sensi Ping', 'Seven of Nine', 'Shama', 'Shamoo', 'Shelob', 'Shen', 'Shira', 'Shortcake', 'Sif', 'Sir Prancalot', 'Sisko', 'Skittles', 'Skully', 'Small Fry', 'Smaoug', 'Smoosh Magoosh', 'Snacks', 'Snitch', 'Snorkel', 'Snowpiercer', 'Sondhiem', 'Sonic', 'Spam', 'Spanks', 'Special #4', 'Speckle', 'Spittle', 'Spout', 'Starbuck', 'Starksky', 'Starlord', 'Stephen', 'Stephen Colbert', 'Sterling', 'Steve Austin', 'Steve Banjo', 'Sting', 'Stormfly', 'Stromboli', 'Styx', 'Suchi', 'Sugar Plum', 'Super Star', 'Synth', 'T-Rex', 'T.S Elio t', 'Taco', 'Taft', 'Tails', 'Tallulah', 'Tangerine', 'Taquito', 'Tasty', 'Tater Tot', 'Teflon', 'Terrasque', 'Terry Gross', 'Tesseract', 'Thinknottle', 'Thor', 'Tiger', 'Tigger', 'Tina', 'Tom ', 'Tom Bodett', 'Tom Bombadill', 'Tom O Mahley', 'Tommy', 'Tony', 'Tony Orlando', 'Toothless', 'Topsy', 'Tormond', 'Totoro', 'Tottsie Roll', 'Trail Mix', 'Trebble', 'Tuk Tuk', 'Tunks', 'Tux', 'Twig', 'Twinkie', 'Two Cat Shapurr', 'Typhoon', 'Tyr', 'Tyrion', 'Tyvek', 'Una', 'Vader', 'Valdimir', 'Vanna White', 'Veggimite', 'Virginia', 'Waffle', 'Walden', 'Warf', 'Wasabi Pea', 'Wednesday', 'WHAM!', 'Whimsy', 'Whisker River', 'Wiernerbutt', 'William Wingfield III', 'Wilson', 'Winn Dixie', 'Won Ton', 'Woody', 'Wooster', 'Wopper', 'Wyvern', 'X Files', 'Zander', 'Zeus', 'Zoot'];
```

Comment:
Maybe (definitely) overkill for this, but it would be awesome if the name list were stored separately from the code. But it would allow for better re-use. Probably a good future-feature.

Reply:
Agreed. I figure build a bicycle before the car though. I'm curious if squarespace can host arbitrary CSV files. If so, then it's just a matter of using fetch or something.

Reply:
Yeah, you can upload files to Squarespace and get permanent URLs for them.